src/System.Security.Principal/System.Security.Principal.sln

```diff
@@ -1,12 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22823.1
+VisualStudioVersion = 14.0.24720.0
```

Comment:
Does the VS version actually matter for anything impactful?  I keep reverting this change whenever VS does it, but it's so insisting, I wouldn't mind just letting it have its way :)